revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
+}
+
+.header-img-div {
+    width: 100%;
+    height: 500px;
+    position: absolute;
+    background-size: cover;
+
+    .overlay {
+        width: 100%;
+        height: 100%;
+        top: 0;
+        left: 0;
+        background: rgba(0, 0, 0, .5);
+        position: absolute;
+        
+        .donate-button-header {
+            border-color: white;
+            border-width: 2px;
+            border-style: solid;
+            width: 15%;
+            height: 10%;
+            top: 410px;
+            left: 80%;
+            z-index: 100;
+            position: absolute;
+            transition: all 0.3s;
+
+            .donate-link {
+                width: 100%;
+                height: 100%;
+                display: inline-block;
+                
+                p {
+                    width: 100%;
+                    height: 100%;
+                    margin: 5% 0px;
+                    color: white;
+                    text-align: center;
+                    transition: all 0.3s;
+                }
+            }
+
+            &:hover {
+                background-color: white;
+
+                p {
+                    color: black;
+                    font-weight: 500;
+                }
+            }
+        }
+
+        .project-title {
+            color: white;
+            font-size: 2rem;
+            font-weight: 800;
+            top: 400px;
+            left: 5%;
+            position: absolute;
+            width: 30%;
+            line-height: 2rem;
+
+        }
+    }
+}
+
+.dashboard {
+    height: 400px;
+    z-index: 1000;
+    position: absolute;
+    top: 300px;
+
+    .graphics-container {
+        height: 100%;
+        width: 100%;
+
+        .top-bar {
+            background-color: $revolv-blue;
+            width: 100%;
+            height: 40px;
+            position: absolute;
+            top: 50%;
+            display: inline-block;
+            margin: 0;
+
+            .container {
+                position: relative;
+                color: white;
+                font-weight: 500;
+                width: 100%;
+                height: 100%;
+                margin: 0;
+                display: inline-block;
+
+                .amount-donated {
+                    float: left;
+                    width: 30%;
+                    height: 100%;
+                    line-height: 40px;
+                    margin: 0px 0px 0px 20px;
+                    font-size: 1.4rem;
+                }
+
+                .days-left {
+                    float: right;
+                    height: 100%;
+                    line-height: 40px;
+                    font-size: 1.4rem;
+                    margin: 0px 20px 0px 0px;
+                }
+            }
+        }
+
+        .bottom-bar {
+            background-color: #064f7e;
+            width: 100%;
+            height: 40px;
+            position: absolute;
+            top: 60%;
+            display: inline-block;
+
+            .container {
```

Comment:
Looks like a lot of styles are repeated between top and bottom bar. Can we make their definitions more DRY? (DRY = Don't Repeat Yourself)